contributing_to_docs/doc_guidelines.adoc

```diff
@@ -42,24 +38,41 @@ Please set your editor to strip trailing whitespace.
 
 [NOTE]
 ====
-The topic title, which is the first line of the document, is the only level 1 ( = ) title. Section headers within the topic must be level 2 ( == ) or lower.
+The topic title, which is the first line of the document, is the only level 1 ( = ) title.
+Section headers (modules) within the topic/assembly must be level 2 ( == ) or lower, by using leveloffsets in include statements.
 ====
 
-*All titles must have a unique (within the OpenShift docs suite) section anchor*, and this anchor must be all lowercase letters, with no line spaces between the anchor and the section title:
+*All modules included in assemblies must have a top level anchor of the format*:
```

Comment:
I'd make a separate section for the module metadata format. I'd also include the fact that "The \`<module_anchor>\` must match the module's file name."